modules/ossm-jaeger-service-mesh.adoc

```diff
@@ -0,0 +1,16 @@
+////
+This CONCEPT module included in the following assemblies:
+-assembly.adoc
+////
+
+[id="ossm-jaeger-service-mesh_{context}"]
+= Jaeger and service mesh
+
+Installing Jaeger via the Service Mesh on OpenShift differs from community Jaeger installations in multiple ways.  These modifications are sometimes necessary to resolve issues, provide additional features, or to handle differences when deploying on OpenShift or OKD.
+
+* Jaeger has been enabled by default.
+* Ingress has been enabled by default.
+* The name for the Zipkin port name has changed to jaeger-collector-zipkin (from http)
+* Jaeger uses Elasticsearch for storage by default.
+* The community version of Istio will inject the sidecar by default if you have labeled the namespace. You are not required to label the namespace with {ProductName}. However, {ProductName} requires that you opt-in to having the sidecar automatically injected to a deployment. This is to avoid injecting a sidecar where it is not wanted (for example build or deploy pods).
```

Comment:
I think this point is talking about the envoy proxy sidecar, not jaeger.

Reply:
Yeah - we just need to be clear it is talking about the Istio sidecar and not the Jaeger agent sidecar.

Reply:
I think this is a potential confusion area. "Sidecar" is a pattern, where an extra container is injected into the pod. Istio injects a proxy sidecar, and Jaeger injects an agent sidecar. Those sidecars aren't the same. When using Istio's sidecar in the context of Jaeger, it's probably best to always say "Proxy sidecar" or "Istio's sidecar". When talking about Jaeger's sidecar, it's better to say "[Jaeger] agent sidecar".

The proxy sidecar creates spans related to the pod's ingress and egress traffic, while the agent sidecar will receive the spans emitted by the application and send to the Jaeger Collector.